src/main/java/frc/robot/util/ConversionUtils.java

```diff
@@ -81,6 +81,14 @@ public static double RPMToFalcon(double RPM, double gearRatio) {
         return motorRPM * (2048.0 / 600.0);
     }
 
+    public static double RPMToMeters(double RPM, double circumference, double gearRatio){
+        return falconToMeters(RPMToFalcon(RPM,gearRatio),circumference,gearRatio);
```

Comment:
This currently calculates wheel RPM to meters. It might be more useful to calculate motor RPM to meters, which means you should use a gear ratio of 1 for the RPMToFalcon conversion.

Reply:
Also, why are you converting a speed to a distance?

Reply:
I named it wrong. I don't use these anyway, so I deleted them. This whole class has a bunch of stuff wrong with it.